app-react/src/App.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+
+import bear from './images/avatars/bear.svg';
+import butterfly from './images/avatars/butterfly.svg';
+import elephant from './images/avatars/elephant.svg';
+import giraffe from './images/avatars/giraffe.svg';
+import goldfish from './images/avatars/goldfish.svg';
+import horse from './images/avatars/horse.svg';
+import octopus from './images/avatars/octopus.svg';
+import parrot from './images/avatars/parrot.svg';
+import Header from './components/Header';
+
+const avatars = [
+  bear,
+  butterfly,
+  elephant,
+  giraffe,
+  goldfish,
+  horse,
+  octopus,
+  parrot,
+];
+
+export default class App extends Component {
```

Comment:
This looks really good! I wonder if we could convert this to a functional component and use React Hooks to manage state? No need to do this now, we can raise another ticket for this if you'd like.